tests/test_metric_libraries.py

```diff
@@ -1,51 +1,102 @@
+        ConfigLoader.get_metric("disable_pocket_clicks", "firefox_desktop"),
+        ConfigLoader.get_metric("disable_pocket_spocs_clicks", "firefox_desktop"),
+    ]
 
-def test_imported_ok():
-    assert mmd.active_hours
-    assert mmf.uri_count
-    assert mmios.baseline_ping_count
-    assert mmfoc.metric_ping_count
-    assert iosfoc.metric_ping_count
-    assert mmk.baseline_ping_count
-    assert iosk.baseline_ping_count
+    fenix_metrics = [
+        ConfigLoader.get_metric("uri_count", "fenix"),
+        ConfigLoader.get_metric("user_reports_site_issue_count", "fenix"),
+        ConfigLoader.get_metric("user_reload_count", "fenix"),
+        ConfigLoader.get_metric("baseline_ping_count", "fenix"),
+        ConfigLoader.get_metric("metric_ping_count", "fenix"),
+        ConfigLoader.get_metric("first_run_date", "fenix"),
+    ]
 
+    other_metrics = []
+    for platform in [
+        "firefox_ios",
+        "focus_android",
+        "focus_ios",
+        "klar_android",
+        "klar_ios",
+    ]:
+        other_metrics += [
+            ConfigLoader.get_metric("baseline_ping_count", platform),
+            ConfigLoader.get_metric("metric_ping_count", platform),
+            ConfigLoader.get_metric("first_run_date", platform),
+        ]
 
-@pytest.fixture()
-def included_metrics():
-    return enumerate_included((mmd, mmf, mmios, mmk, mmfoc), Metric)
+    return desktop_metrics + fenix_metrics + other_metrics
 
 
 @pytest.fixture()
 def included_datasources():
-    return enumerate_included((mmd, mmf, mmios, mmk, mmfoc), DataSource)
+    "Returns a thorough but not necessarily exhaustive list of data sources"
+    desktop_datasources = [
+        ConfigLoader.get_data_source("clients_daily", "firefox_desktop"),
+        ConfigLoader.get_data_source(
+            "search_clients_engines_sources_daily", "firefox_desktop"
+        ),
+        ConfigLoader.get_data_source("main_summary", "firefox_desktop"),
+        ConfigLoader.get_data_source("events", "firefox_desktop"),
+        ConfigLoader.get_data_source("normandy_events", "firefox_desktop"),
+        ConfigLoader.get_data_source("main", "firefox_desktop"),
+        ConfigLoader.get_data_source("crash", "firefox_desktop"),
+        ConfigLoader.get_data_source("cfr", "firefox_desktop"),
+        ConfigLoader.get_data_source("activity_stream_events", "firefox_desktop"),
+    ]
+
+    other_datasources = []
+    for platform in [
+        "fenix",
+        "firefox_ios",
+        "focus_android",
+        "focus_ios",
+        "klar_android",
+        "klar_ios",
+    ]:
+        other_datasources += [
+            ConfigLoader.get_data_source("baseline", platform),
+            ConfigLoader.get_data_source("events", platform),
+            ConfigLoader.get_data_source("metrics", platform),
+        ]
+
+    return desktop_datasources + other_datasources
 
 
 def test_sql_not_detectably_malformed(included_metrics, included_datasources):
```

Comment:
Does this test mean that mozanalysis is testing the validity of definitions from metric-hub? Or is this testing something about mozanalysis's sql generation? I'd think the former should be the responsibility of metric-hub to test, though they're only testing on merges, not continuously.

Reply:
Yeah I was wondering about that.  These tests are essentially testing the functionality of configloader, so the question is whether or not these give us anything the tests in `test_config` don't?

Reply:
And if they are doing something useful we should maybe just put them in `test_config.py`

Reply:
Looking it over all this is covered in `test_config.py`, I vote we delete `test_metric_librarires.py` and `test_segement_libraries.py`